unison-cli/src/Unison/Codebase/Editor/Input.hs

```diff
@@ -112,10 +111,10 @@ data Input
     | ResolveTypeNameI Path.HQSplit'
   -- edits stuff:
     | LoadI (Maybe FilePath)
-    | AddI [HQ'.HashQualified Name]
-    | PreviewAddI [HQ'.HashQualified Name]
-    | UpdateI (Maybe PatchPath) [HQ'.HashQualified Name]
-    | PreviewUpdateI [HQ'.HashQualified Name]
+    | AddI (Set Name)
+    | PreviewAddI (Set Name)
+    | UpdateI (Maybe PatchPath) (Set Name)
```

Comment:
I don't think you ever could do that, since the only time you'd know the hash is if it was conflicted, and the version of slurping on trunk doesn't allow resolving conflicts with an `update`.

I think in the future we'll probably want to allow _some_ version of resolving conflicts this way though 😄 , probably just by running an _update_  on a term which is conflicted.